ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/server/computation/RecoveryTest.java

```diff
@@ -105,6 +106,9 @@
   @Mock
   private Errors errorHandler = mock(Errors.class);
 
+  @Mock
```

Comment:
nit: unused

Reply:
Removed in https://github.com/confluentinc/ksql/pull/8480/commits/7ac1fe2c1b271aec0498c7f1315b34f0681bb7a4#diff-48e54df909b0174a4501759566450a4f8988e57588e9b4dc0ff56999dfab88aa.